src/batou/lib/git.py

```diff
@@ -27,13 +27,20 @@ class Clone(Component):
     target = "."
     revision = None
     branch = None
+    tag = None
     vcs_update = True
     clobber = False
 
     def configure(self):
-        if (not self.revision_or_branch) or (self.revision and self.branch):
+        if (
```

Comment:
I think this would be easier to to write down as:

```
def exactly_one(self, *args):
    return sum(map(bool, args)) == 1

if exactly_one(self.revision, self.branch, self.tag):
    pass
```

Reply:
Nice and readable idea, thanks!

Reply:
Done.